Dockerfile

```
@@ -41,9 +41,9 @@ ARG GIT_COMMIT
 ARG GIT_TREE_STATE
 
 RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build \
-      -ldflags "-w -X ${VERSION_PACKAGE}.version=${VERSION} -X ${VERSION_PACKAGE}.buildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ') -X ${VERSION_PACKAGE}.gitCommit=${GIT_COMMIT} -X ${VERSION_PACKAGE}.gitTreeState=${GIT_TREE_STATE}" \
-      -o bin/kargo \
-      ./cmd/controlplane \
+    -ldflags "-w -X ${VERSION_PACKAGE}.version=${VERSION} -X ${VERSION_PACKAGE}.buildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ') -X ${VERSION_PACKAGE}.gitCommit=${GIT_COMMIT} -X ${VERSION_PACKAGE}.gitTreeState=${GIT_TREE_STATE}" \
+    -o bin/kargo \
+    ./cmd/controlplane \
```

```suggestion
      -ldflags "-w -X ${VERSION_PACKAGE}.version=${VERSION} -X ${VERSION_PACKAGE}.buildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ') -X ${VERSION_PACKAGE}.gitCommit=${GIT_COMMIT} -X ${VERSION_PACKAGE}.gitTreeState=${GIT_TREE_STATE}" \
      -o bin/kargo \
      ./cmd/controlplane \
```

Just avoiding incidental changes.

---

Auto-formatting the Dockerfile keeps correcting this, maybe it's worth keeping?

---

This doesn't matter a _ton_ one way or the other. Consider this suggestion take it or leave it.

I'm curious though what is auto-formatting the Dockerfile that way for you. I use the Docker plugin for VS Code and unless I just have it misconfigured, it doesn't do any auto-formatting for me. It would be a useful thing to have.

---

I'm using the same VSCode + Docker extension `ms-azuretools.vscode-docker`, and I have my Code configured to format on save, that's all. I'll leave it as you suggested though, I'm done with the changes. :)

```json
"editor.formatOnSave": true
```

---

TIL. Thanks for that!